specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json

```diff
@@ -485,7 +477,12 @@
             }
           }
         }
-      }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
```

Comment:
just `$ref` `../../../../common-types/resource-management/v1/types.json#/definitions/Resource` here, then you can remove the definition of `Resource` above (which is also slightly invalid since you `$ref` the entire file "types.json" up there, but really you just want to extract `/definitions/Resource` from that file) 🙂